packages/vm/src/state/stateManager.ts

```diff
@@ -544,7 +544,7 @@ export default class DefaultStateManager implements StateManager {
    */
   async hasGenesisState(): Promise<boolean> {
     const root = this._common.genesis().stateRoot
-    return await this._trie.checkRoot(root)
+    return await this._trie.checkRoot(toBuffer(root))
```

Comment:
Just so that this is not slipping through: can you give a short comment on the nature respectively potential effect of this change? Is this just some more-concrete-typing adoption? Or might this also have fixed up-till-now hiddedn bugs?